easy-ui-tokens/src/size/icon.json

```diff
@@ -4,7 +4,8 @@
       "xs": { "value": "16px", "attributes": { "unit": "rem" } },
       "sm": { "value": "20px", "attributes": { "unit": "rem" } },
       "md": { "value": "24px", "attributes": { "unit": "rem" } },
-      "lg": { "value": "32px", "attributes": { "unit": "rem" } }
+      "lg": { "value": "32px", "attributes": { "unit": "rem" } },
+      "xl": { "value": "56px", "attributes": { "unit": "rem" } }
```

Comment:
**Question:**
Did you confirm with design that we won't be seeing any sizes between 32px and 56px? Seems like quite the jump and I worry a 48px one will come along soon...

Reply:
i agree. i updated with intermediate values since according to design the current constraint is simply "multiples of 8" with no known upper bound. with that, i'm not sure how exactly it'll evolve. i think we'll need to keep an eye on it and adjust accordingly as the design system grows/gets use